src/simple_lottery/wrapper.rs

```diff
@@ -0,0 +1,36 @@
+use super::algorithm;
+use super::init::make_setup;
+use super::params::Params;
+use super::proof::Proof;
+use super::setup::Setup;
+use crate::utils::types::Element;
+
+/// The main simple lottery struct with prove and verify functions.
+#[derive(Debug, Clone, Copy)]
+pub struct Wrapper {
+    setup: Setup,
+}
+
+impl Wrapper {
+    /// Initialize ALBA with `Params`.
+    pub fn create(params: &Params) -> Self {
+        let setup = make_setup(params);
+        Self::create_unsafe(&setup)
+    }
+
+    /// This function is unsafe to use and should be avoided.
```

Comment:
```suggestion
    /// Use this function with caution, e.g. in tests or with trusted `Setup`.
```

Reply:
Let's do this in #65. I copied this comment from `src/centralized_telescope/wrapper.rs`.